polynote-frontend/polynote/ui.js

```diff
@@ -710,9 +710,10 @@ export class NotebookUI extends UIEventTarget {
         this.editBuffer = new EditBuffer();
 
         this.cellUI.addEventListener('UpdatedConfig', evt => {
+            console.log("updating config")
             const update = new messages.UpdateConfig(path, this.globalVersion, ++this.localVersion, evt.detail.config);
             this.editBuffer.push(this.localVersion, update);
-            this.socket.send(this.localVersion, update);
```

Comment:
I think this was the only thing causing the Dependencies bug that @faisalzs ran into...

Reply:
😮 what kind of idiot.... *git blame*... er, I mean... good catch! 😅